addons/binding/org.openhab.binding.bigassfan/ESH-INF/thing/thing-types.xml

```diff
@@ -98,17 +129,32 @@
 			</options>
 		</state>
 	</channel-type>
-	<channel-type id="fan-speed-min">
+	<channel-type id="fan-learn-speed-min">
```

Comment:
Yes, that's the terminology they use to describe setting the min and max speed when using Smarter Cooling.